server/deploy.rb

```diff
@@ -0,0 +1,40 @@
+# Other useful stuff
+require 'fileutils'
+require 'paint'
+
+class Deploy
```

Comment:
Should this be changed to something a bit more clear? Like `UploadDeploy` or `DeployServer` or something like that? `Deploy` seems a bit broad

Reply:
Naming is always a challenge...

I tried to pick something that speaks to the name of the object itself. I'm hesitant against to use `UploadDeploy` as that speaks to the action of the call and not the function of the consumer ( your service ) IMO. 

What about `CreateRelease` or `BuildRelease` since every new payload rise sees is unique given a uuid ? 

Thoughts ?

Reply:
`CreateRelease` is good. I just wanted something a bit more specific than `Deploy`